digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/lane/LaneFiller.scala

```diff
@@ -232,7 +232,7 @@ class LaneFiller {
     def SegmentPieceCreation(startM: Double, endM: Double, lanes: Seq[PersistedLane]): Seq[SegmentPiece] = {
 
       lanes.filter(lane => lane.startMeasure <= startM && lane.endMeasure >= endM)
-           .map( x => SegmentPiece(x.id, x.laneCode, startM, endM, SideCode(x.sideCode), x.attributes) )
+           .map( x => SegmentPiece(x.id, startM, endM, SideCode(x.sideCode), x.attributes) )
```

Comment:
Since this method is inside of combine function and this segmentPieces created inside the combinator seems to be returned as a Seq[PersistedLane] to be created.
Shouldn't they need to have laneCode to be create correctly?
Because when a measure change it will not expire and create a new lane?

Reply:
That part of code was rollbacked from a previous attempt of fixing a bug that led to another bug unrelated to the issue you posted.That issue you are talking about wasn't and isn't happening.

Reply:
My concern is if this remove will not make that old bug appear again.
But if you say it is solved, it is solved.